### PR TITLE
Completed css-transitions exercise.

### PR DESCRIPTION
**Pull request by samlee0258**
https://user-images.githubusercontent.com/110208589/217476412-d336af1c-d193-4d8d-8b6d-3a9f939a3793.mp4

### CODE REVIEW

File: css-transitions/styles.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  cursor: pointer;
+  transform: scale(1);
+  transition: 0.5s;
+}
+
+.card > img {
+  width: 100%;
+}
+
+.custom-button:hover {
```
Comment (robgmerrill) [inline css-transitions/styles.css:48]: For code reading clarity pseudoclasses should follow the elements whose state they are responding to. For example: 

```
.custom-button {
}

.custom-button:hover {
}
```

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! I left some feedback on where pseudoclasses should appear in the source order of a CSS file.

File: css-transitions/styles.css (hunk 1 above)
Comment (samlee0258) [reply css-transitions/styles.css]: Screenshot 2023-02-08 at 9 29 32 AM

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**